src/dev/user.clj

```diff
@@ -1,28 +1,62 @@
 (ns user
-  (:require [io.pedestal.http :as http]))
+  (:require
+   [clojure.edn :as end]
+   [io.pedestal.http :as http]
+   [io.pedestal.http.route :as route]
+   [cheffy.routes :as routes]
+   [cheffy.server :as server]
+   [com.stuartsierra.component :as component]))
 
 (println "Loading user.clj")
 
 (defonce ^:private system-ref (atom nil))
 
 (defn start-server []
-  (->> {::http/routes #{}
-        ::http/type :jetty
-        ::http/port 3000
-        ::http/join? false}
-       http/create-server
-       http/start
-       (#(do (println "server started.") %))
-       (reset! system-ref)))
+  (let [config (-> (slurp "src/config/development.edn") end/read-string)]
+    (->> config
+        server/create-system
+        component/start
```

Comment:
Maybe I  could have `server/start`that would combine `create-system` and `start` instead of forcing clients (user.clj, -main) to call `component/start` by themselves?